lib/Conversion/FIRRTLToRTL/LowerToRTL.cpp

```diff
@@ -1468,8 +1460,8 @@ LogicalResult FIRRTLLowering::visitDecl(MemOp op) {
 
     // Now that we have the wires for each element, rewrite any subfields to use
     // them instead of the subfields.
-    while (!port->use_empty()) {
-      auto portField = cast<SubfieldOp>(*port->user_begin());
+    while (!port.use_empty()) {
+      auto portField = cast<SubfieldOp>(*port.user_begin());
```

Comment:
port->user_begin()?

Reply:
I think this one is right? `port` was changed to be the output of `Value MemOp::getPortNamed`, so this should be an actual value now. (Weaker reasoning: it doesn't compile otherwise...)

Reply:
agree that this is right